src/mon/NVMeofGwMap.cc

```diff
@@ -934,6 +939,15 @@ void NVMeofGwMap::get_health_checks(health_check_map_t *checks) const
         ss.str(), gatewayDownDetail.size());
     d.detail.swap(gatewayDownDetail);
   }
+  if (!gatewayInDeletingDetail.empty()) {
+    ostringstream ss;
+    ss << gatewayInDeletingDetail.size() << " gateway(s) are in deleting state"
+      << "; either deploy a new gateway OR "
+      << "move namespaces in it's ANA group ID to another load balancing group.";
```

Comment:
GWs will remain in Deleting state, until the number of namespaces are balanced across the GWs and across the Subsystems. This should be done automatically, and finish in few minutes.